chamber/models.py

```diff
@@ -107,6 +115,13 @@ def eval_props(self):
         self.rho_m = 1 / HAPropsSI('Vha', 'T', ref_temp, 'Y', ref_x,
                                    'P', self.pressure)
 
+        self.nu_m = self.mu_m/self.rho_m
+        self.alpha_m = self.k_m/(self.cp_m * self.rho_m)
+        self.beta_m = 1/ref_temp
+        self.beta_star_m = (const.M2 - const.M1)/(ref_x*const.M1*(const.M2 - const.M1)/(ref_x*const.M1 + (1-ref_x)*const.M2) + const.M1)
+
```

Comment:
We should also try to keep the property evaluation in alphabetical order. Again, this is just to help keep things neat as the code base grows.